### ISSUE TITLE
Consumer offsets not displaying (missing partitions)

### ISSUE THREAD

**Issue by mikesparr** [ldaniels528/trifecta#26]
Kafka 0.10.0 on Ubuntu 14.04.04 LTS 
Trifecta 0.20.0

Using `kafka-python` client I consume a topic with 9 partitions and then view in Trifecta > Inspect > Consumers > {topic} > {group} and it only displays offsets for a portion of the partitions. I expect to see all partitions.

<img width="810" alt="screen shot 2016-07-01 at 3 01 18 pm" src="https://cloud.githubusercontent.com/assets/5553105/16534645/9709323a-3f9e-11e6-8ce1-bdad05f344a8.png">

Using the CLI tool I perform the following and it confirms they were all consumed and no lag:
~~~
ubuntu@db1:~$ kafka-consumer-offset-checker --group etl-diffs --topic rets.nnrmls.Agent --zookeeper localhost:2181

Group           Topic                          Pid Offset          logSize         Lag             Owner
etl-diffs       rets.nnrmls.Agent              0   3175            3175            0               none
etl-diffs       rets.nnrmls.Agent              1   3691            3691            0               none
etl-diffs       rets.nnrmls.Agent              2   3352            3352            0               none
etl-diffs       rets.nnrmls.Agent              3   3432            3432            0               none
etl-diffs       rets.nnrmls.Agent              4   3111            3111            0               none
etl-diffs       rets.nnrmls.Agent              5   2856            2856            0               none
etl-diffs       rets.nnrmls.Agent              6   3247            3247            0               none
etl-diffs       rets.nnrmls.Agent              7   3690            3690            0               none
etl-diffs       rets.nnrmls.Agent              8   3072            3072            0               none
~~~

I upgraded to latest `0.21` and same issue it appears.

<img width="798" alt="screen shot 2016-07-01 at 3 17 39 pm" src="https://cloud.githubusercontent.com/assets/5553105/16534706/1fe40bac-3f9f-11e6-9449-5deb6347e2e2.png">

**Comment by ldaniels528** [comment]
I'll take a look, and get it fixed. :-)

**Comment by ldaniels528** [comment]
Could you confirm whether these issues still exist in the latest release?

**Comment by mikesparr** [comment]
Trying to deploy latest release but paths and directories changed (`trifecta-ui` instead of `trifecta_ui` folder naming, etc) and broke our build scripts. Fixing and will report. Thanks

**Comment by mikesparr** [comment]
Hi @ldaniels528 sorry been working on other projects. I upgraded to rc8 a couple months ago and ever since I cannot view the messages any more.

 * v0.22.0rc8b (0.10.1.0)

<img width="1518" alt="screenshot 2017-05-09 11 08 59" src="https://cloud.githubusercontent.com/assets/5553105/25863275/82823000-34a8-11e7-9fcc-bef15dd6f02b.png">

As such, it's difficult to tell if other issue resolved.

**Comment by mikesparr** [comment]
I'm unsure what changed and I proxy requests to Trifecta via Nginx on web server and haven't had time to troubleshoot. V `0.20` worked fine and then with `0.22.x` file names changed and had to update my deploy scripts and the UI hasn't worked since.

**Comment by likesh1980** [comment]
I agree, the same happening for me as well. It seems like when I have a broker 0.10.1.x, it works fine, but when I have a broker 0.10.0.x, I can not view the messages. I will try with a different version and see if this helps.
